cypress/support/index.d.ts

```diff
@@ -33,6 +33,15 @@ declare namespace Cypress {
      */
     executeQuickAction(actionName: string, modal?: boolean, isDialogExpected ?: boolean): Chainable<any>
 
+    /**
+     * Run a quickAction inside a modal which opens a table on the right side.
+     *
+     * Example: Runs the action "Open HU selection" window inside the window Picking Terminal (Prototype).
+     *
+     * @param actionName - internal name of the action to be executed
+     */
+    executeQuickActionWithRightSideTable(actionName: string): Chainable<any>
+
     /**
      * @param fieldName - name of the field is question
      * @param modal - optional, default = false - use true if the field is in a modal overlay
```

Comment:
if the **component** is in a modal overlay.

Reply:
I think here it's better to leave field, since the function name is also `getString**Field**Value`

Reply:
My bad, I wanted to leae this comment on selectNthRow method not here.